### PR TITLE
[No ticket] Debug KubernetesClusterNotFound in dev

### PR DESCRIPTION
**Pull request by jdcanas**

### CODE REVIEW

File: http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/monitor/NonLeoMessageSubscriber.scala [hunk 1]
```diff
@@ -86,6 +86,7 @@ class NonLeoMessageSubscriber[F[_]: Timer](gkeAlg: GKEAlgebra[F],
     for {
       ctx <- ev.ask
       clusterId = msg.clusterId
+      _ <- logger.info(s"Beginning to delete kubernetes cluster ${clusterId} in project ${msg.project} from NonLeoMessageSubscriber | trace id: ${ctx.traceId}")
```
Comment (rtitle) [inline http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/monitor/NonLeoMessageSubscriber.scala]: We are trying to do `logger.info(ctx.loggingCtx)(msg)` now, so the trace id doesn't need to be embedded in the string.

Comment (kyuksel) [inline http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/monitor/NonLeoMessageSubscriber.scala]:
```suggestion
      _ <- logger.info(ctx.loggingCtx)(s"Beginning to delete kubernetes cluster ${clusterId} in project ${msg.project} from NonLeoMessageSubscriber")
```

**Review by kyuksel — APPROVED** [review]
👍🏼  with one change request inline

File: http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/monitor/NonLeoMessageSubscriber.scala [hunk 2]
```diff
@@ -86,6 +86,7 @@ class NonLeoMessageSubscriber[F[_]: Timer](gkeAlg: GKEAlgebra[F],
     for {
       ctx <- ev.ask
       clusterId = msg.clusterId
+      _ <- logger.info(ctx.loggingCtx)(s"Beginning to delete kubernetes cluster ${clusterId} in project ${msg.project} from NonLeoMessageSubscriber")
```
Comment (Qi77Qi) [inline http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/monitor/NonLeoMessageSubscriber.scala]: not sure if this is necessary. If a cluster is deleted from `zombieMonitor`, we should have `deletedFrom` field indicating that (if cron job is not doing that currently, we should update there)

Comment (Qi77Qi) [reply http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/monitor/NonLeoMessageSubscriber.scala]: oh nvm, cron job just sends message to leo, so I guess we can really differentiate

**Comment by jdcanas** [comment]
multi-test

**Pull request merged**